Microsoft.DotNet.Interactive.PowerShell/Microsoft.DotNet.Interactive.PowerShell.csproj

```diff
@@ -1,5 +1,23 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Target Name="PSRestoreBuiltInModules" AfterTargets="Restore" BeforeTargets="Build">
+    <PropertyGroup>
+      <PSUtilityUrl>https://raw.githubusercontent.com/PowerShell/PowerShell/master/src/Modules/Windows/Microsoft.PowerShell.Utility/Microsoft.PowerShell.Utility.psd1</PSUtilityUrl>
+      <PSManagementUrl>https://raw.githubusercontent.com/PowerShell/PowerShell/master/src/Modules/Windows/Microsoft.PowerShell.Management/Microsoft.PowerShell.Management.psd1</PSManagementUrl>
```

Comment:
Is there a versioned, published source we can use so that the build is more deterministic and reproducible?

Reply:
Or at least point it to the master now using the permalink url.
Also this is coming from the ```Windows``` folder. is it platform specific?

Reply:
> Or at least point it to the master now using the permalink url.

I will use a link that is pinned to the latest release tag.

Unfortunately, the powershell built-in modules are not published today, and yes @colombod, the manifest file is platform dependent :(
We just have to live with this solution for now, but I opened https://github.com/PowerShell/PowerShell/issues/11783 to address this in our nuget packages.